crates/ruff_linter/src/rules/refurb/helpers.rs

```diff
@@ -39,31 +41,45 @@ pub(super) fn generate_method_call(name: Name, method: &str, generator: Generato
     generator.stmt(&stmt.into())
 }
 
-/// Format a code snippet comparing `name` to `None` (e.g., `name is None`).
-pub(super) fn generate_none_identity_comparison(
-    name: Name,
+/// Returns a fix that replace `range` with
+/// a generated `a is None`/`a is not None` check.
+pub(super) fn replace_with_identity_check(
```

Comment:
Is my understanding correct that this was mainly moved from `isinstance_type_none` with the exception that you added the comments check?